### ISSUE TITLE
RFE: reschedule playlist

### ISSUE THREAD

**Issue by echarlie** [wuvt/pload#23]
it should be possible to reschedule a playlist in the "edit playlist" view.

**Comment by mutantmonkey** [comment]
Yeah, I'm actually thinking a calendar-like view that shows playlists and supports dragging and dropping would be better as well. We may want a separate issue for that, but it will support this feature.

**Comment by echarlie** [comment]
I think first-cut will involve just an "edit playtime" in the playlist editor, where it lets you tweak all the same forms as when you create the playlist (perhaps as a modal)
